packages/help-center/src/hooks/use-should-render-email-option.tsx

```diff
@@ -1,12 +1,13 @@
 import { useSupportAvailability } from '@automattic/data-stores';
 
 export function useShouldRenderEmailOption() {
-	const { data: supportAvailability } = useSupportAvailability( 'OTHER' );
-	if (
-		supportAvailability?.is_user_eligible_for_tickets ||
-		supportAvailability?.is_user_eligible_for_upwork
-	) {
-		return true;
-	}
-	return false;
+	const { data: supportAvailability, isFetching } = useSupportAvailability( 'OTHER' );
+
+	return {
+		isLoading: isFetching,
+		render:
+			( supportAvailability?.is_user_eligible_for_tickets ||
+				supportAvailability?.is_user_eligible_for_upwork ) ??
+			false,
```

Comment:
Just FYI, this is my go to method for forcing booleans on null checks.

```suggestion
			Boolean( supportAvailability?.is_user_eligible_for_tickets ||
				supportAvailability?.is_user_eligible_for_upwork )
```